syntax = "proto3";

package NetProto;

message StreamHeader {
  fixed32 message_id = 1;
  fixed32 message_length = 2;
}
